experiments/azure/cyclecloud/cpu/experiment/data/quicksilver/slurm-quicksilver.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
```

Comment:
Ditto here - is there a reason slurm submission scripts are under data here and not configs?

Reply:
They should be removed, and are now.